web/src/enterprise/campaigns/global/create/CreateCampaign.tsx

```diff
@@ -41,7 +41,7 @@ export const CreateCampaign: React.FunctionComponent<Props> = ({ className }) =>
                         <div>
                             <h3>Track existing changesets</h3>
                             <p className="mb-0">
-                                Track a collection of already created changesets by adding them to a campaign. The
+                                Track a collection of already created changesets by collecting them to a campaign. The
```

Comment:
```suggestion
                                Track a collection of already created changesets by collecting them in a campaign. The
```

not a 1000% sure about this one but the other sounded odd

Reply:
No, you're 1000% spot on :)